src/Files.App/Helpers/DriveHelpers.cs

```diff
@@ -136,9 +136,13 @@ public static Data.Items.DriveType GetDriveType(System.IO.DriveInfo drive)
 			};
 		}
 
-		public static async Task<StorageItemThumbnail> GetThumbnailAsync(StorageFolder folder)
-			=> (StorageItemThumbnail)await FilesystemTasks.Wrap(()
-				=> folder.GetThumbnailAsync(ThumbnailMode.SingleItem, 40, ThumbnailOptions.UseCurrentScale).AsTask()
+		public static async Task<StorageItemThumbnail?> GetThumbnailAsync(StorageFolder folder)
+		{
+			return folder is null
+				? null
+				: (StorageItemThumbnail)await FilesystemTasks.Wrap(()
+					=> folder.GetThumbnailAsync(ThumbnailMode.SingleItem, 40, ThumbnailOptions.UseCurrentScale).AsTask()
 			);
+		}
```

Comment:
This is an unrelated fix.

Reply:
I added it because I was getting that exception every time I started Files and it was annoying. Anyway I'll remove it if you think so

Reply:
This will be fixed by #12286.